README.md

```diff
@@ -1,53 +1,34 @@
-# Edit your Shopify theme locally
 
-[![Gem Version](https://badge.fury.io/rb/shopify_theme.svg)](http://badge.fury.io/rb/shopify_theme)
-
-The Shopify theme gem is a command line tool that lets you make live changes to themes on your Shopify store. If the command line is scary, check out the [Desktop Theme Editor app](http://apps.shopify.com/desktop-theme-editor).
-
-It will watch your local folders for any changes in your theme (including adding and removing files) and will update your .myshopify.com store to the latest changes. 
+# Edit Your Shopify Theme Locally
 
 ![Shopify theme gem](https://dl.dropboxusercontent.com/u/669627/terminalreadme.png)
 
-You do not need to make changes to your default theme. You can leverage the theme preview feature of Shopify
-that allows you to view what an unpublished theme looks like on your Shopify store. This means you don't need to
-sign up for extra accounts and keep your shop up to date. You will however have a blue bar that shows up that you can
-remove via the web inspector in Chrome or Safari.
-
-# Requirements
+The Shopify theme gem is a command line tool that allows you to manage and edit theme files directly on your computer. This allows you to use your preferred code editing software opposed to the built in code editor available in the “Edit HTML/CSS” page of your store. Files can be manually or automatically uploaded, downloaded, and synced. Depending on how you develop themes and your personal preference, this can allow for a more enjoyable and streamlined workflow.  If the command line is scary for you, check out the [Desktop Theme Editor app](http://apps.shopify.com/desktop-theme-editor).
 
-This gem works with OS X or Windows with Ruby 1.9. 
 
-First time installing Ruby on windows? Try [Rubyinstaller](http://http://rubyinstaller.org/). 
+## Requirements
+This gem works with OS X or Windows with Ruby 1.9.
+First time installing Ruby on windows? Try Rubyinstaller.
```

Comment:
This might make more sense as unordered list or separate them by extra newlines. It's all showing up on one line through the magic of markdown.